nw-firestore-schema.ts

```diff
@@ -33,6 +33,17 @@ interface WebsiteData {
   }
 }
 
+interface LivesiteData {
+  [key]: any;
+  //eg
+  links: Link[];
+}
+
+interface Link {
+  label: string;
+  href: string;
+}
+
 interface DayOfEvent {
```

Comment:
can we change the time field to be a Date object as the start, and a duration in like minutes

Reply:
Oh do we need duration of workshops as well?

Reply:
thoughts on making it startTime and endTime

Reply:
yeah that works too

Reply:
might screw up existing code so will need to make sure ppl know we are changing the schema

Reply:
do the schemas support class hierarchies? thoughts on making a base `Event` interface which has a name, start, and end time, and everything else can extend that

Reply:
🤷 @Ciuffi ?

Reply:
The schema system supports class hierarchies, its not a bad idea.

Also, we are storing it as a string because that is how its stored in firebase. This isn't the js interpretation, its the firebase one.

(btw, we store dates/times as UTC ISO strings just to make it easier.